lib/parser/token-parsers.h

```diff
@@ -704,10 +704,22 @@ constexpr auto rawName{nonDigitIdChar >> many(nonDigitIdChar || digit)};
 TYPE_PARSER(space >> sourced(rawName >> construct<Name>()))
 constexpr auto keyword{construct<Keyword>(name)};
 
+constexpr auto logicalTRUE{
+    (".TRUE."_tok ||
+        extension<LanguageFeature::LogicalAbbreviations>(".T."_tok)) >>
+    pure(true)};
+constexpr auto logicalFALSE{
+    (".FALSE."_tok ||
+        extension<LanguageFeature::LogicalAbbreviations>(".F."_tok)) >>
+    pure(false)};
```

Comment:
If the LogicalAbbreviations language feature is enabled, we should also report an error if the user tries to define an operator named ".T." or ".F.", right? I happen to be working on that right now.

I'm happy to see this feature is off by default.

Reply:
That is correct (although I did not test it).  This is one of the few extensions that we cannot enable by default without putting conforming programs at risk (unless we start tracking defined `OPERATOR(.foo.)` names as state in the parser).